docs/Cli.md

```diff
@@ -133,6 +133,7 @@ Re-apply any new defaults as desired.
 | `auto_disarm_delay`             |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | 0      | 60     | 5             | Master       | UINT8    |
 | `small_angle`                   | If the copter tilt angle exceed this value the copter will refuse to arm. default is 25°.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | 0      | 180    | 25            | Master       | UINT8    |
 | `pid_at_min_throttle`           | If enabled, the copter will process the pid algorithm at minimum throttle.  Cannot be used when `retarded_arm` is enabled.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | OFF    | ON     | ON            | Master       | UINT8    |
+| `airmode_saturation_limit`      | When air mode is enabled this will act as a "crash detection". It prevents too much compensation for very high errors what can cause spazzing out. 0 = no limit 100 is always limited during saturation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | 0      | 100    | 50            | Master       | UINT8    |
```

Comment:
(new issue may be adequate)
So there is gyro overflow? ..  Where? 
Gyro chip should return 13bit signed value (-4096-4095) with saturation and overflow indication. 
Rest of code may use saturation logic arithmetic to handle overflow ...

Also there may be problem with gyro saturation - it may cause PID i-term growing when command is larger that gyro measurement range. Constraining commanded input within gyro range can help